aspnetcore/security/app-secrets.md

```diff
@@ -86,23 +76,17 @@ You can also use the Secret Manager tool to list, remove and clear app secrets.
 
 ## Accessing user secrets via configuration
 
-You access Secret Manager secrets through the configuration system. Add the `Microsoft.Extensions.Configuration.UserSecrets` as a dependency in your *project.json* file and run `dotnet restore`.
+You access Secret Manager secrets through the configuration system. Add the `Microsoft.Extensions.Configuration.UserSecrets` to the *.csproj* file and run `dotnet restore`.
```

Comment:
Add the `NameOfPackage` package
or 
Add `NameOfPackage`

But more importantly, I thought we were going to use file-aagnostic language for installing packages, so couldn't this just be Install the X package?